docs/monitors/openshift-cluster.md

```diff
@@ -0,0 +1,141 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# openshift-cluster
+
+Collects cluster-level metrics from the
+Kubernetes API server.  It uses the _watch_ functionality of the K8s API
+to listen for updates about the cluster and maintains a cache of metrics
+that get sent on a regular interval.
+
+Since the agent is generally running in multiple places in a K8s cluster and
+since it is generally more convenient to share the same configuration across
+all agent instances, this monitor by default makes use of a leader election
+process to ensure that it is the only agent sending metrics in a cluster.
+All of the agents running in the same namespace that have this monitor
+configured will decide amongst themselves which should send metrics for this
+monitor, and the rest will stand by ready to activate if the leader agent
+dies.  You can override leader election by setting the config option
+`alwaysClusterReporter` to true, which will make the monitor always report
+metrics.
+
+This monitor is similar to
+[kube-state-metrics](https://github.com/kubernetes/kube-state-metrics), and
+sends many of the same metrics, but in a way that is less verbose and better
+fitted for the SignalFx backend.
```

Comment:
Can you add a note about how it is related to the `kubernetes-cluster` monitor and how they *don't* need to run both?  I can imagine people seeing both and thinking they complement each other instead of being exclusive.